main.go

```diff
@@ -129,15 +130,20 @@ func main() {
 	if err != nil {
 		if exitError := new(exec.ExitError); errors.As(err, &exitError) {
 			exitCode := exitError.ExitCode()
+			if exitCode == -1 {
```

Comment:
I'm thinking about we could check if Rspec exit code is not 1 (Rspec exits due to test failure) before entering the code block for retry. Maybe this would give us more confidence the retry command gets executed only when there's test failure?

```
if exitCode != 1 {
	logErrorAndExit(exitCode, "%s exited abnormally, cannot continue", testRunner.Name())
}
```

Reply:
I've updated this so instead of returning if the exit code is `-1`, it will return early (it'll still exit with -1 though) if a signal terminated the runner process.